Assets/Models/DungeonPieces.FBX.meta

```diff
@@ -1,7 +1,7 @@
 fileFormatVersion: 2
```

Comment:
When doing a fresh import, I see the warning:
File 'DungeonPieces' has inconsistent LOD naming (_LOD1 found but no _LOD0).

Reply:
Gotcha, let me rename things